<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnswerRouteSearch</class>
 <widget class="QWidget" name="AnswerRouteSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: rgb(39, 100, 54);
font: 36pt &quot;Oswald&quot;;
border-bottom: 1px solid #000000;</string>
   </property>
   <property name="text">
    <string>Recommended Route</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="BottomFill">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>510</y>
     <width>800</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: rgb(39, 100, 54);
font: 36pt &quot;Oswald&quot;;
border-bottom: 1px solid #000000;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Back">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>412</y>
     <width>340</width>
     <height>60</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: white;
	background-color: rgb(39, 100, 54);
	font: 18pt &quot;Oswald&quot;;
	border-radius: 20px;
	border: 1px solid #000000;
}

QPushButton::Hover{
	border-color: rgb(0, 0, 0);
	border: 5px solid #000000;
}

QPushButton::Pressed{
	border-color: rgb(0, 0, 0);
	border: 5px solid;
	background-color: rgb(90,152,101);
}
</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="Text1">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>127</y>
     <width>510</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: white;
	background-color: rgb(39, 100, 54);
	font: 18pt &quot;Oswald&quot;;
	border-radius: 20px;
	border: 1px solid #000000;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Text2">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>222</y>
     <width>510</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: white;
	background-color: rgb(39, 100, 54);
	font: 18pt &quot;Oswald&quot;;
	border-radius: 20px;
	border: 1px solid #000000;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Text3">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>317</y>
     <width>510</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: white;
	background-color: rgb(39, 100, 54);
	font: 18pt &quot;Oswald&quot;;
	border-radius: 20px;
	border: 1px solid #000000;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
